### ISSUE TITLE
Tongue Weakness

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10571]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 34053870 Late-onset Pompe disease (LOPD) is an inherited progressive metabolic myopathy that presents in the first year of life to adulthood. Clinical presentation is heterogeneous, differential diagnosis is challenging, and diagnostic delay is common. One challenge to differential diagnosis is the overlap of clinical features with those encountered in other forms of acquired/hereditary myopathy. **Tongue weakness** and imaging abnormalities are increasingly recognized in LOPD.

PMID: 20301305 Oculopharyngeal muscular dystrophy (OPMD) is characterized by ptosis and dysphagia due to selective  involvement of the muscles of the eyelids and pharynx, respectively. Early manifestations of dysphagia include  increased time needed to consume a meal and an acquired avoidance of dry foods. The severity of dysphagia is the major determinant of prognosis, as it leads to potentially life-threatening aspiration pneumonia and poor nutrition. Other manifestations, observed as the disease progresses, are limitation of upward gaze, **tongue atrophy and weakness**, chewing difficulties, wet voice, facial muscle weakness, axial muscle weakness, proximal  limb girdle weakness predominantly in lower limbs, and proximal upper extremity weakness.

**Parent term (use hpo.jax.org/app)**
Abnormal tongue physiology HP:0030810 (but I'm not sure)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:420429 Glycogen storage disease due to acid maltase deficiency, late-onset
ORPHA:270 Oculopharyngeal muscular dystrophy

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
The current term Difficulty in tongue movements HP:0000183 is being used to annotate diseases characterized by weakness of the tongue muscle and in one case by Tongue spasticity and in another case by Ankyloglossia.
I will change the label of the term to 
Tongue muscle weakness and reannotate the two diseases.

**Comment by pnrobinson** [comment]
The term is

HP:0000183
Tongue muscle weakness

**Comment by carolina-f** [comment]
Thanks, I think it's more appropriate
